pkg/crc/constants/constants.go

```diff
@@ -200,6 +202,8 @@ func GetDefaultMemory(preset crcpreset.Preset) int {
 	switch preset {
 	case crcpreset.OpenShift:
 		return 9216
+	case crcpreset.OKD:
```

Comment:
could've been 
`case crcpreset.OpenShift, crcpreset.OKD` since both have the same CPU and memory values

Reply:
@anjannath ye we can do that also, I just wanted to explicit